hexpat/HexPat.g4

```diff
@@ -0,0 +1,598 @@
+grammar HexPat;
```

Comment:
Please include a license (e.g., MIT), and copyright notice. There is no overarching license and copyright for the repo.

Reply:
Please also include a readme.md file. You can copy one from one of the other grammars and adjust accordingly. The readme helps clarify authorship, source of the grammar (e.g., a spec), where the corresponding tools that implement the language elsewhere, and so on. Thanks!!